listenbrainz/webserver/views/art_api.py

```diff
@@ -101,21 +107,49 @@ def cover_art_grid_post():
     if err is not None:
         raise APIBadRequest(err)
 
-    if not isinstance(r["release_mbids"], list):
-        raise APIBadRequest("release_mbids must be a list of strings specifying release_mbids")
-
-    for mbid in r["release_mbids"]:
-        if not is_valid_uuid(mbid):
-            raise APIBadRequest(f"Invalid release_mbid {mbid} specified.")
-
     if "cover_art_size" in r:
         cover_art_size = r["cover_art_size"]
     elif r["image_size"] < 1000:
         cover_art_size = 250
     else:
         cover_art_size = 500
 
-    images = cac.load_images(r["release_mbids"], tile_addrs=tiles, layout=layout, cover_art_size=cover_art_size)
+    # Get release_mbids
+    release_mbids = []
+    if "release_mbids" in r:
+        if not isinstance(r["release_mbids"], list):
+            raise APIBadRequest("release_mbids must be a list of strings specifying release_mbids")
+
+        release_mbids = list(r["release_mbids"])
+
+    # Get release_group_mbids
```

Comment:
I think we don't need to support having both release mbids and release group mbids. If release mbids are specified used them if not try for release group mbids.